uiowa/uiowa
# Facilities - a fix for encoded aerial links for button render

# To Test

- Sync facilities on main
- Add a named building aerial link http://maps.facilities.uiowa.edu/php/ipa.php?lat=41.659258&lon=-91.548093&ori=w
- Click the rendered button link. See that it is littered with `amp;` and that the page does not full resolve.
- Checkout branch.
- Run cim, cr.
- Click the button link again. ??? Profit.
